tests/post_training/test_templates/helpers.py

```diff
@@ -47,19 +52,18 @@ def __len__(self) -> int:
         return self._len
 
 
-def get_static_dataset(
-    input_size: Tuple,
-    transform_fn: Callable,
-    fn_to_type: Callable,
-) -> Dataset:
+def get_static_dataset(input_size: Tuple, transform_fn: Callable, fn_to_type: Callable, length: int = 1) -> Dataset:
```

Comment:
The description wasn't updated with the new param.

Reply:
done